### DISCUSSION TITLE
Does the API contain any open source software?

### DISCUSSION THREAD

**Discussion by nwebb89** [weather-gov/api#565] [General]
I tried searching around for this answer but I've been asked to confirm if any open source software is used for this. 

Thanks!!

**Comment by scadergit** [comment]
Until there is approvial to convert the API to open-source (we have discussed it to allow community contribution), I would not be at liberty to discuss implementation details. Let us know if you have a specific concern.

**Reply by nwebb89** [reply]
Is there any licensing agreement governing this api? This is the concern I'm being asked right now.

**Reply by scadergit** [reply]
The API is public use within the rate limits established for the service. The helpdesk would better describe those details.
